src/dropdown.test.interactions.multiple.ts

```diff
@@ -1506,20 +1544,154 @@ it('updates its tags when a selected option is disabled programmatically', async
       <glide-core-dropdown-option
         label="Two"
         value="two"
+        disabled
         selected
       ></glide-core-dropdown-option>
     </glide-core-dropdown>`,
   );
 
-  const option = host.querySelector('glide-core-dropdown-option');
-  assert(option);
+  const options = host.querySelectorAll('glide-core-dropdown-option');
 
-  option.disabled = true;
+  assert(options[1]);
+  options[1].disabled = false;
   await host.updateComplete;
 
   const tagContainers = host.shadowRoot?.querySelectorAll<HTMLElement>(
     '[data-test="tag-container"]',
   );
 
-  expect(tagContainers?.length).to.equal(1);
+  expect(host.value).to.deep.equal(['one', 'two']);
+  expect(tagContainers?.length).to.equal(2);
+});
+
+it('sets `aria-selected` when an option is selected programmatically', async () => {
```

Comment:
Because Dropdown Option now takes into account whether its the last selected option when setting `aria-selected`, I had to move these tests from [`dropdown.option.test.interactions.single.ts`](https://github.com/CrowdStrike/glide-core/pull/929/files#diff-f15854470b7bcd5c3a2d447a97ecdbfbc31175c568fbcdc48d70ab625ce9e591) because it didn't make sense for an isolated Dropdown Option test suite to test multiple Dropdown Options in a Dropdown.